<launch>
  <node pkg="phoenix_navigation" type="rail_navigation.py" name="rail_navigation" output="screen">
    <rosparam command="load" file="$(find phoenix1_parameters)/config/rail_positions.yaml" subst_value="true" />
    <param name="map_file_name" value="$(find phoenix_navigation)/maps/sim_map/sim_map" />
  </node>
</launch>

